src/test/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposerTest.java

```diff
@@ -0,0 +1,452 @@
+            .setField(Field.builder().setName("content").setType(TypeNode.STRING).build())
+            .build();
+    TypeNode severityType =
+        TypeNode.withReference(
+            VaporReference.builder().setName("Severity").setPakkage(PACKAGE_NAME).build());
+    MethodArgument arg2 =
+        MethodArgument.builder()
+            .setName("severity")
+            .setType(severityType)
+            .setField(
+                Field.builder().setName("severity").setType(severityType).setIsEnum(true).build())
+            .build();
+    List<List<MethodArgument>> signatures = Arrays.asList(Arrays.asList(arg1, arg2));
+    Method unaryMethod =
+        Method.builder()
+            .setName("echo")
+            .setMethodSignatures(signatures)
+            .setInputType(inputType)
+            .setOutputType(outputType)
+            .build();
+    String results =
+        SampleCodeWriter.write(
+            ServiceClientSampleCodeComposer.composeUnaryRpcMethodSampleCode(
+                unaryMethod, signatures.get(0), clientType, resourceNames));
+    String expected =
+        "try (EchoClient echoClient = EchoClient.create()) {\n"
+            + "  String content = \"content951530617\";\n"
+            + "  Severity severity = Severity.forNumber(0);\n"
+            + "  EchoResponse response = echoClient.echo(content, severity);\n"
+            + "}";
+    assertEquals(expected, results);
+  }
+
+  @Test
+  public void composeUnaryRpcMethodSampleCode_methodReturnVoid() {
+    TypeNode inputType =
+        TypeNode.withReference(
+            VaporReference.builder()
+                .setName("DeleteUserRequest")
+                .setPakkage("com.google.showcase.v1beta1")
+                .build());
+    TypeNode outputType =
+        TypeNode.withReference(
+            VaporReference.builder().setName("Empty").setPakkage("com.google.protobuf").build());
+    List<List<MethodArgument>> methodSignatures =
+        Arrays.asList(
+            Arrays.asList(
+                MethodArgument.builder()
+                    .setName("name")
+                    .setType(TypeNode.STRING)
+                    .setField(Field.builder().setName("name").setType(TypeNode.STRING).build())
+                    .build()));
+    Method unaryMethod =
+        Method.builder()
+            .setName("delete")
+            .setMethodSignatures(methodSignatures)
+            .setInputType(inputType)
+            .setOutputType(outputType)
+            .build();
+    String results =
+        SampleCodeWriter.write(
+            ServiceClientSampleCodeComposer.composeUnaryRpcMethodSampleCode(
+                unaryMethod, methodSignatures.get(0), clientType, resourceNames));
+    String expected =
+        "try (EchoClient echoClient = EchoClient.create()) {\n"
+            + "  String name = \"name3373707\";\n"
+            + "  echoClient.delete(name);\n"
+            + "}";
+    Assert.assertEquals(results, expected);
+  }
+}
```

Comment:
One more obscure edge case to test: The case where the method has no arguments. 

I recently learned this case actually exists. Please see [cloud_billing.proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/billing/v1/cloud_billing.proto#L55) and the relevant client class in [java-billing](https://github.com/googleapis/java-billing).

Reply:
Good to know. Added this edge case.